113 add japanese to reset password

Shingo023 opened the pull request:
### Overview

1. Added Japanese language support to the Forgot Password and Password Reset pages.
2. Updated the email verification flow: when a verification link has expired, customers are now required to log in to receive a new one. Previously, the verification email was sent automatically, but for improved security, customer identification via login is now required.

### Review Preparation

1. Set your browser language to Japanese.
2. Run `npm run build` in the `/backend` directory.
3. Visit the customer login page: http://localhost:3000/customers/login.

### Review Points
Note: The logic part of this PR has already been reviewed, so code review is not required.
1. Try logging in with the following credentials and confirm that the message shown in the image below appears, as the dummy customer's email has not been verified:
- Email: `kiv-developers@googlegroups.com`
- Password: `AaasoBo!Hana`
![スクリーンショット 2025-05-30 午前8 14 23](https://github.com/user-attachments/assets/9e3297d3-f1b9-4678-afc6-6618b17a249a)

2. In Prisma Studio, set the `expires` value of the `VerificationToken` to the previous day and save the change. Then, check the inbox and click the 「メールアドレスを認証」 button in the verification email. Confirm that the message in the image below appears.
![スクリーンショット 2025-05-30 午前8 15 16](https://github.com/user-attachments/assets/a4bd8837-5ef5-4448-b527-4bf326e4fdbf)
![スクリーンショット 2025-05-30 午前8 16 17](https://github.com/user-attachments/assets/d5f39381-5caf-4362-8672-8d44d4e4aa0a)
![スクリーンショット 2025-05-30 午前8 17 15](https://github.com/user-attachments/assets/a214bb6a-c096-43be-bbb2-282d19950935)

3. Go back to the login page and click 「パスワードを忘れた場合」. Enter `kiv-developers@googlegroups.com` and click the 「送信」 button. Confirm that the message in the image below appears.
![スクリーンショット 2025-05-30 午前8 22 12](https://github.com/user-attachments/assets/2d0985aa-5b30-44ce-aff1-e5176054f606)

4. In Prisma Studio, set the `expires` value of the `PasswordResetToken` to the previous day and save the change. Then, check the inbox and click the 「パスワードを再設定」 button in the reset email. Enter a new password, click the 「送信」 button, and confirm that the message in the image below appears.
![スクリーンショット 2025-05-30 午前8 26 10](https://github.com/user-attachments/assets/513b97e3-6996-46e0-9a03-2cca0f888e4e)
![スクリーンショット 2025-05-30 午前8 23 09](https://github.com/user-attachments/assets/45de0c5b-91fe-4fb2-ab9c-eab59ab0a431)
![スクリーンショット 2025-05-30 午前8 27 26](https://github.com/user-attachments/assets/d3096565-1839-4395-8c6f-86ef7a160114)

5. After requesting a new link, check the inbox again and click the 「パスワードを再設定」 button in the most recent reset email. Enter a new password, click the 「送信」 button, and confirm that the message in the image below appears.
![スクリーンショット 2025-05-30 午前10 37 31](https://github.com/user-attachments/assets/f4f8dd54-dd2b-4a3d-b4f3-2bf2dddb9fbc)

mtkksk1780 reviewed (approved):
Thank you for the implementation! This feature is very helpful for customers. I have confirmed the behavior, and everything worked as you described. I would like to share two points I noticed during the check.

#### [Review point 4]
It might be more user-friendly to show the expired message before entering a new password.
<img width="800" alt="Screenshot 2025-06-01 at 16 24 53" src="https://github.com/user-attachments/assets/90f8dc71-5f4e-451d-a2a4-edd6582bf8bc" />
<img width="800" alt="Screenshot 2025-06-01 at 16 25 10" src="https://github.com/user-attachments/assets/3c05d81c-27be-4c0a-99ed-41c145c0de63" />

#### [Review point 5]
I was able to confirm that point 5 itself worked without any issues. However, after that, I wasn't able to log in to the app using the new password. This might be outside the scope of this PR, but I would like to share it for the record.
<img width="800" alt="Screenshot 2025-06-01 at 16 28 41" src="https://github.com/user-attachments/assets/77a14866-8403-41c0-b625-fd64b8c02fe3" />

mtkksk1780 commented on frontend/src/app/helper/api/adminsApi.ts:
```diff
@@ -143,7 +143,7 @@ export const registerAdmin = async (userData: {
 
     if (response.status === 409) {
-      return { email: EMAIL_ALREADY_REGISTERED_ERROR };
+      return { email: EMAIL_ALREADY_REGISTERED_ERROR.en };
```
I tried to check the point, but I realized the admins' registration page was broken.

<img width="1123" alt="Screenshot 2025-06-01 at 17 16 07" src="https://github.com/user-attachments/assets/b8e2315b-52b2-48fc-8a42-566567867d5d" />

This is because the file was updated in #104. Until then, the first three items (`name`, `password`, and `passwordStrength`) had been shared between customers and admins, but the structure was changed in that PR.
Could you update the file to display the necessary information on the admins' registration page? 

[PR104:`frontend/src/app/components/features/registerForm/RegisterForm.tsx`]
https://github.com/KivSystemAdmin/aaasobo-management-system/pull/104/files#diff-20dabe59b13570669e6e9162282f61e5f74df34c9ade4fad6d3321a414f3503b

Shingo023 commented:
@mtkksk1780 

Thank you so much for your helpful feedback, and I’m really sorry for the inconvenience!

> [Review point 4]
> It might be more user-friendly to show the expired message before entering a new password.

That was an insightful point! I’ve updated the code so that an error message is now shown before prompting users to enter a new password when the token is invalid or expired.
Could you please confirm review point 4 again? The message shown in the screenshot below should now appear.
![スクリーンショット 2025-06-03 午前10 55 57](https://github.com/user-attachments/assets/684eb5e4-e9f8-4dd4-b256-7104d4838fc1)

Also, if you click the reset button in one of the old emails, a different message should be displayed as show below.
![スクリーンショット 2025-06-03 午前10 55 12](https://github.com/user-attachments/assets/478fe3ea-5f0d-4165-a27d-77ae8d405e87)

> [Review point 5]
> I was able to confirm that point 5 itself worked without any issues. However, after that, I wasn't able to log in to the app using the new password. This might be outside the scope of this PR, but I would like to share it for the record.

Thank you! That’s expected — once you click the button in the email verification email, you should be able to log in with your new password.

> Could you please take care of the following? After that, I will check what you asked me!

I'm so sorry for the inconvenience! I've fixed `RegisterForm.tsx`, and you should now be able to register a new admin from the admin dashboard.
![スクリーンショット 2025-06-03 午前11 30 40](https://github.com/user-attachments/assets/b660502a-0bd8-4dc9-8492-c087e76c418a)

mtkksk1780 reviewed (approved):
Thank you for the update! I have rechecked the behavior, and all points are covered, including the admin registration feature!
